src/evaluation_class.py

```diff
@@ -813,48 +811,14 @@ def enumerate_tasks(self, max_instance_count: int, **kwargs):
                             'output': oracle_action_sequence
                         })
 
-                # get the input values from the web page
-                inputs_with_values = self.extract_values(inputs)
-
                 # collecting field statistics
                 if task_name not in results:
                     results[task_name] = {}
 
-                # TODO: use my score_outputs function
-                score = 0.0
-                for i in inputs_with_values:
-                    if i.name in self.excluded_input_names:
-                        continue
+                score = self.score_outputs(inputs, answers_map, results[task_name])
```

Comment:
@klxu03 Notice I replaced this segment with our existing evaluation function `score_outputs` so we have less repetition.